buildenv/jenkins/JenkinsfileBase

```diff
@@ -25,12 +25,12 @@ def makeTest(testParam) {
 def setupEnv() {
 	env.JOBSTARTTIME = sh(script: 'LC_TIME=C date +"%a, %d %b %Y %T %z"', returnStdout: true).trim()
 
-	// Check what existing java processes are running
-	echo "PROCESSCATCH: Java processes which are currently on the machine:"
-	if (PLATFORM.contains("windows")) {
-		sh(script:'powershell -c "Get-Process java | select id,processname,starttime"', returnStatus:true)
-	} else {
-		sh(script:'ps -fu '+"${env.USER}"+' | grep java | egrep -v "remoting.jar|agent.jar|grep"', returnStatus: true)
+	// Terminate any previous test related processes that are running
+	echo "PROCESSCATCH: Terminating previous test related processes which are currently on the machine:"
+	def statusCode=sh(script:"aqa-tests/terminateTestProcesses.sh 2>&1", returnStatus:true)
+	echo "returnCode = ${statusCode}"
+	if (statusCode != 0) {
+		echo "Unable to terminate all processes."
```

Comment:
Should we just call `terminateJavaProcesses()`?

Reply:
yeah :-)